finalPrep/modZeroHero.js

```diff
@@ -72,13 +129,58 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
 
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name;
+    this.superpower = superpower;
+    this.age = age;
+  }
+
+  static archNemesis = "The Syntax Error";
```

Comment:
I think you want `archNemesis`, `powerLevel`, and `energyLevel` to be declared in the constructor as `instance variables` rather than `static variables` - using the keyword `static` allows variables or methods to be associated with the `class` rather than the instance of that class.